.github/workflows/build-timestamped-master.yml

```diff
@@ -38,7 +38,10 @@ jobs:
                     packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
                     publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
                     publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                run: ./gradlew publish --no-daemon --scan
+                run: |
+                    git config user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                    git config user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
```

Comment:
Any use case to add to master/pull request?

Reply:
This was just a test :D Wanted to test how this works out.

Reply:
Ack. On second thought, I think we need this for pull requests. However, it needs to be the user invoking the pull request and not ballerina bot

Reply:
Yeah, that's what I am looking for. Seems like there's no straightforward way to get the user email inside a workflow.

Reply:
@niveathika Do we actually need the git user in the pull-request though? I think we don't.

We only have to commit this while a release process, right? In other cases, it will be committed in the local builds. (And the post-relese PR will commit the files in most cases. So anyways the user does not have to commit it?

Reply:
Ack